src/Integration.Vsix/CFamily/CLangAnalyzer.cs

```diff
@@ -176,8 +176,15 @@ private void RunAnalysis(IRequest request, IIssueConsumer consumer, IAnalysisSta
                 }
                 else
                 {
-                    var analysisTime = DateTime.Now - analysisStartTime;
-                    statusNotifier?.AnalysisFinished(request.Context.File, messageHandler.IssueCount, analysisTime);
+                    if (messageHandler.AnalysisSucceeded)
```

Comment:
The CLangAnalyzer needed to be updated to report "analysis failed" for any internal error.